linode_api4/objects/database.py

```diff
@@ -0,0 +1,405 @@
+from linode_api4.objects import Base, Property, MappedObject, DerivedBase, UnexpectedResponseError
+
+
+class DatabaseType(Base):
+    api_endpoint = '/databases/types/{id}'
+
+    properties = {
+        'deprecated': Property(filterable=True),
+        'disk': Property(),
+        'engines': Property(),
+        'id': Property(identifier=True),
+        'label': Property(),
+        'memory': Property(),
+        'vcpus': Property(),
```

Comment:
You should note the additional property in a comment, like we do [here](https://github.com/linode/linode_api4-python/blob/ca858c7d9416e0aeed92e34a7f449f5e513d26c5/linode_api4/objects/linode.py#L147)

```suggestion
        'vcpus': Property(),
        # type_class is populated from the 'class' attribute of the returned JSON
```